pkg/gitreceive/build.go

```diff
@@ -183,62 +138,94 @@ func build(conf *Config, s3Client *s3.S3, builderKey, gitSha string) error {
 		return fmt.Errorf("uploading %s to %s/%s (%v)", absAppTgz, bucketName, tarObjKey, err)
 	}
 
+	creds := storage.CredsOK()
+
+	var pod *api.Pod
+	var buildPodName string
+	if usingDockerfile {
+		buildPodName = dockerBuilderPodName(appName, shortSha)
+		pod = dockerBuilderPod(
+			conf.Debug,
+			creds,
+			buildPodName,
+			conf.PodNamespace,
+			appConf.Values,
+			tarURL,
+			imageName,
+		)
+	} else {
+		buildPodName = slugBuilderPodName(appName, shortSha)
+		pod = slugbuilderPod(
+			conf.Debug,
+			creds,
+			buildPodName,
+			conf.PodNamespace,
+			appConf.Values,
+			tarURL,
+			pushURL,
+			buildPackURL,
+		)
+	}
+
 	log.Info("Starting build... but first, coffee!")
 	log.Debug("Starting pod %s", buildPodName)
-	kCreateCmd := exec.Command(
-		"kubectl",
-		fmt.Sprintf("--namespace=%s", conf.PodNamespace),
-		"create",
-		"-f",
-		finalManifestFileLocation,
-	)
-	if log.IsDebugging {
-		kCreateCmd.Stdout = os.Stdout
-	}
-	kCreateCmd.Stderr = os.Stderr
-	if err := run(kCreateCmd); err != nil {
+	log.Debug("Pod spec: %v", pod)
```

Comment:
Will this output something reasonably human-readable? If not, can you write something that will generate a cleaner-looking string?